plutus-tx-plugin/test/Plugin/Debug/9.6/letFun.pir.golden

```diff
@@ -8,19 +8,19 @@
       { no-src-span }
       (datatype
         { no-src-span }
-        (tyvardecl { no-src-span } Bool-445 ({ no-src-span } type))
+        (tyvardecl { no-src-span } Bool-451 ({ no-src-span } type))
```

Comment:
I thought we didn't end up printing uniques for golden files, but apparently we did.